components/stimulation/StimulationStudioRepeatDelayModal.vue

```diff
@@ -111,8 +115,15 @@ export default {
       return metric_label;
     },
   },
+  watch: {
+    is_valid() {
+      if (this.is_valid) this.is_enabled_array = [true, true, true];
+      if (!this.is_valid) this.is_enabled_array = [false, true, true];
```

Comment:
Could maybe shorten this up with `this.is_enabled_array = [this.is_valid, true, true];